packages/storybook/stories/doc-navigator.tsx

```diff
@@ -0,0 +1,59 @@
+import { useOf } from '@storybook/blocks';
+import { type ModuleExports } from '@storybook/types';
+import { navigate } from '@storybook/addon-links';
+import { OdsButton, OdsLink } from '@ovhcloud/ods-components-react';
+import { ZeroheightLinkIds } from './zeroheight-link-ids';
+import React from 'react';
+
+export const DocNavigator = ({ of, children }: { of: ModuleExports, children: React.ReactNode }) => {
+  const resolvedOf = useOf(of || 'story', ['meta']);
+  const demoStoryKeys = Object.keys(resolvedOf.csfFile.stories || {}).filter(storyKey => storyKey.includes('--demo'));
+
+  const extractLinkId = (storyId: string): string | null => {
+    for (const id of Object.values(ZeroheightLinkIds)) {
+      const idPart = id.split('-');
+      const componentPart = idPart.slice(1).join('-');
+      if (storyId.includes(componentPart)) {
+        return id;
+      }
+    }
+    return null;
+  };
+
+  return (
+    <div style={{ display: 'flex', justifyContent: 'center', alignItems: 'center' }}>
+      <div>
+        {demoStoryKeys.map((storyId, index) => (
+          <OdsButton
+            key={index}
+            label={'Go to Demo'}
+            variant={'default'}
+            onClick={() => {
+              navigate({ storyId })
+            }}
+          >
+          </OdsButton>
+        ))}
+      </div>
+      <div style={{ marginLeft: '20px' }}>
+        <OdsLink
+          label={'Go to Zeroheight'}
```

Comment:
Maybe a less tool-oriented label? (Go to functional specification?)
BTW for fixed string you can omit the `{}` (`label="label"`)